rate: 100 # [Hz]

offset_x: 465710.759073
offset_y: 5249465.47126
